streams/nz.m3u

```diff
@@ -7,8 +7,6 @@ https://juicex.nz/hls/mystream.m3u8
 https://ptvlive.kordia.net.nz/out/v1/3fc2254c865a457c8d7fbbce227a2aae/index.m3u8
 #EXTINF:-1 tvg-id="ParliamentTV.nz" status="online",Parliament TV (1080p)
 https://ptvlive.kordia.net.nz/out/v1/daf20b9a9ec5449dadd734e50ce52b74/index.m3u8
-#EXTINF:-1 tvg-id="Three.nz" status="blocked",Three [Geo-blocked]
-https://livestreamdirect-three.mediaworks.nz/three.m3u8
```

Comment:
Possibly needs header and Geo-blocked.

Reply:
The report said that it does not load even in NZ: https://github.com/iptv-org/iptv/issues/10171. I had the same thing with my NZ VPN, it didn't load.